migration-engine/migration-engine-tests/tests/migrations/diff.rs

```diff
@@ -117,7 +117,7 @@ fn from_empty_to_migrations_directory(mut api: TestApi) {
 
     let expected_printed_messages = expect![[r#"
         [
-            "-- CreateTable\nCREATE TABLE \"cats\" (\n    \"id\" INTEGER NOT NULL,\n    \"moos\" BOOLEAN DEFAULT false,\n\n    CONSTRAINT \"cats_pkey\" PRIMARY KEY (\"id\")\n);\n",
+            "-- CreateSchema\nCREATE SCHEMA IF NOT EXISTS \"prisma-tests\";\n\n-- CreateTable\nCREATE TABLE \"prisma-tests\".\"cats\" (\n    \"id\" INTEGER NOT NULL,\n    \"moos\" BOOLEAN DEFAULT false,\n\n    CONSTRAINT \"cats_pkey\" PRIMARY KEY (\"id\")\n);\n",
```

Comment:
I can't pinpoint where the prefixing of the table names is coming from. IIRC it was a point of discussion with Jan and I think we want to keep the output of migrate in single-schema mode (when no schemas array is specified in the datasource) pretty much the same as now, e.g. no schema name.

Reply:
Let's try to keep this working without schema names if you don't have multiple schemas specified. Maybe we can clear the namespaces vec if multi schema mode is used.